test/src/test/java/org/corfudb/runtime/collections/CorfuStoreShimTest.java

```diff
@@ -104,6 +109,58 @@ public void checkDirtyReads() throws Exception {
         log.debug(table.getMetrics().toString());
     }
 
+    @Test
+    public void checkIndexes() throws Exception {
```

Comment:
can we rename the test to something more explanatory?  something like testSecondaryIndexes()

Reply:
done